packages/story-editor/src/components/canvas/canvasLayout.js

```diff
@@ -67,6 +67,16 @@ function CanvasLayout({ header, footer }) {
   useLayoutParams(backgroundRef);
   const layoutParamsCss = useLayoutParamsCssVars();
 
+  const { pageWidth, pageHeight } = useLayout(
+    ({ state: { pageWidth, pageHeight } }) => ({
+      pageWidth,
+      pageHeight,
+    })
+  );
+
+  // If we don't have proper canvas dimensions yet, don't bother rendering element layers.
+  const hasDimensions = pageWidth !== 0 && pageHeight !== 0;
```

Comment:
Nit: If either of those is `0`, probably doesn't make sense to render? Although not sure if that ever happens.

Reply:
Yeah doesn't really make sense to render in that case; which is why this checks for both to be non-zero.

Reply:
Ah, right, it is already checking for that 💡 Misread the code.